src/elements/about-block.html

```diff
@@ -0,0 +1,170 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/iron-icon/iron-icon.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="hoverboard-icons.html">
+
+<dom-module id="about-block">
+    <template>
+        <style is="custom-style" include="shared-styles flex flex-alignment flex-reverse"></style>
+
+        <style>
+             :host {
+                display: block;
+                position: relative;
+                color: var(--dark-primary-color);
+            }
+
+            .container {
+                margin: 48px 40px;
+            }
+
+            .description-block,
+            .statistics-block {
+                padding: 16px;
+            }
+
+            .statistics-block .row-1 {
+                margin-bottom: 16px;
+            }
+
+            .statistics-block .row-2 {
+                margin-top: 16px;
+            }
+
+            .block-cta paper-button {
+                padding-left: 0;
+                margin-left: 0;
+            }
+
+            .block-cta iron-icon {
+                margin-left: 8px;
+            }
+
+            .block-cta .cta-label {
+                font-size: 14px;
+            }
+
+            .block-subtitle {
+                color: var(--secondary-text-color);
+            }
+
+            .icon-label {
+                color: var(--secondary-text-color);
+                margin-top: 8px;
+            }
+
+            .count-info {
+                padding: 16px 0;
+            }
+
+            .numbers {
+                margin-left: 16px;
+                font-size: 32px;
+                vertical-align: middle;
+            }
+
+            .icon iron-icon {
+                color: var(--dark-primary-color);
+            }  
+
+            @media (min-width: 868px) {               
+                .description-block {
+                    margin-right: 16px;
+                    padding: 48px;
+                }                
+
+                .statistics-block {
+                    margin-left: 16px;
+                }
+
+                 .numbers {
+                    font-size: 50px;
+                }                
+            }
+
+             @media (min-width: 1150px) {
+                  .description-block,
+                  .statistics-block {
+                    padding: 48px 64px;
+                    margin: 0 48px;
+                }
+             }
+
+        </style>
+
+        <div class="container" flex layout horizontal center wrap-reverse>
+
+            <div class="description-block" flex>
+                <h2 class="title">{$ aboutBlock.title.partA $}</br>{$ aboutBlock.title.partB $}</h3>
```

Comment:
max-width wouldn't work when the width becomes smaller. Better just to set width.